specification/app/resource-manager/Microsoft.App/preview/2022-01-01-preview/ContainerApps.json

```diff
@@ -551,6 +610,141 @@
           "type": "boolean"
         }
       }
+    },
+    "CustomDomain": {
+      "description": "Custom Domain of a Container App",
+      "type": "object",
+      "properties": {
+        "name": {
+          "description": "Hostname.",
+          "type": "string"
+        },
+        "bindingType": {
+          "description": "Custom Domain binding type.",
+          "enum": [
+            "Disabled",
+            "SniEnabled",
+            "IpBasedEnabled"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "bindingType",
+            "modelAsString": false
+          }
+        },
+        "virtualIP": {
+          "description": "Virtual IP address assigned to the hostname if IP based binding is enabled.",
+          "type": "string"
+        },
```

Comment:
Is this relevant for containerapps?

Reply:
Once we support the "reservedIP" feature, it will be used. It is not supported today (and I don't think it will be for GA)

Reply:
If it is not going to be in GA then let's remove it from here. Otherwise public api client SDKs will have this property that does nothing.

Reply:
Removed VirtualIP